addons/phantom_camera/scripts/managers/phantom_camera_manager.gd

```diff
@@ -18,6 +18,10 @@ var phantom_camera_3ds: Array: ## Note: To support disable_3d export templates f
 		return _phantom_camera_3d_list
 var _phantom_camera_3d_list: Array ## Note: To support disable_3d export templates for 2D projects, this is purposely not strongly typed.
 
+func _ready():
+	if Engine.has_singleton(PHANTOM_CAMERA_CONSTS.PCAM_MANAGER_NODE_NAME):
+		Engine.unregister_singleton(PHANTOM_CAMERA_CONSTS.PCAM_MANAGER_NODE_NAME)
```

Comment:
Does this need to be unregistered first?
Wonder if it could just be a case of adding the singleton if it doesn't exist — basically, inverting the conditional checker and adding the `Engine.register_singleton` inside of it.